src/data/danish-gigaword.py

```diff
@@ -0,0 +1,237 @@
+ year = 2021,
+ booktitle = {Proceedings of the 23rd Nordic Conference on Computational Linguistics},
+ publisher = {NEALT}
+}
+"""
+
+_DESCRIPTION = """\
+ The Danish Gigaword Corpus contains raw text spanning several different domains and forms. The dataset is available under the Creative Commons Attribution-ShareAlike
+ License.
+"""
+_HOMEPAGE = "https://gigaword.dk"
+_LICENSE = "Creative Commons Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)"
+_DATA_URL = "https://bit.ly/danishgigaword10"
+
+
+_INDIVIDUAL_LICENSES = {
+    "adl": cc0,
+    "botxt": cc0,
+    "cc": cc0,
+    "danavis": cc0,
+    "dannet": dannet_license,
+    "datwitter": twitter_license,
+    "depbank": att_sharealike_4,
+    "ep": cc0,
+    "ft": cc0,
+    "gutenberg": gutenberg_license,
+    "hest": cc0,
+    "jvj": att_sharealike_4,
+    "naat": cc0,
+    "opensub": opensub_license,
+    "relig": cc0,
+    "retsinformationdk": retsinformationdk_license,
+    "retspraksis": cc0,
+    "skat": cc0,
+    "spont": cc0,
+    "synne": cc0,
+    "tv2r": tv2r_license,
+    "twfv19": twitter_license,
+    "wiki": cc0,
+    "wikibooks": cc0,
+    "wikisource": cc0,
+}
+
+
+class DAGWConfig(datasets.BuilderConfig):
+    """BuilderConfig for DAGW."""
+
+    def __init__(self, data_url, **kwargs):
+        """BuilderConfig for DAGW
+
+        Args:
+          data_url: `string`, url to the dataset
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(DAGWConfig, self).__init__(
+            version=datasets.Version(
+                "1.0.0",
+            ),
+            **kwargs,
+        )
+        self.data_url = data_url
+
+
+class DAGW(datasets.GeneratorBasedBuilder):
+    """The Danish Gigaword Corpus contains raw text spanning several different domains and forms. The dataset is available under the Creative Commons Attribution-ShareAlike
+    License."""
+
+    VERSION = datasets.Version("0.1.0")
+    BUILDER_CONFIGS = [
+        DAGWConfig(
+            name="DAGW-v1",
+            data_url=_DATA_URL,
+            description="Document level dataset. Each row contains one document (of greatly varying length)",
+        )
+    ]
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            # This is the description that will appear on the datasets page.
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "text": datasets.Value("string"),
+                    "source": datasets.Value("string"),
+                    "doc_id": datasets.Value("string"),
+                    "LICENSE": datasets.Value("string"),
```

Comment:
det her kan vel sagtens være en kategorisk? Så at teksten ikke fylder for hvert felt?